src/select/select.js

```diff
@@ -76,17 +77,19 @@ class Select extends React.Component<PropsT, StatelessStateT> {
   }
 
   componentDidMount() {
-    /* eslint-disable-next-line cup/no-undef  */
-    document.addEventListener('click', this.handleClickEvent, {
-      capture: true,
-    });
+    if (__BROWSER__) {
```

Comment:
Is this check needed for linting?

Reply:
Following convention in other components where we only attach DOM events when on the browser. Though looking at it now, I think `componentDidMount` is only called on client-side anyway so maybe we can forgo this check here?